modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -1949,6 +1954,11 @@ private CacheArguments parseAndValidateCacheArgs() {
                         cacheArgs.dump(true);
                     else if (CMD_SKIP_ZEROS.equals(nextArg))
                         cacheArgs.skipZeros(true);
+                    else if (CACHE_KIND.equals(nextArg)) {
+                        String kind = nextArg("Cache kind should be assigned").toUpperCase();
```

Comment:
I think the list of possible values will be helpful here. For example? something like the following:
`The cache kind should be specified. The following values can be used: ALL, SYSTEM, PERSISTENT, NOT_PERSISTENT.`

Reply:
Done.